docs/api/README.md

```diff
@@ -223,3 +224,55 @@ A string that any included selectors in `rules` will be appended to. Use to scop
   />
 </div>
 ```
+
+## Print Style
+
+In order to fully support all print styling eventually, it is necessary to use CSS because of browser differences described in #132. Radium allows you to do this easily by specifying print styles as static properties of your components.
```

Comment:
"In order to fully support print styling it is necessary to use CSS because of browser differences, as described in #132. Radium allows you to do this easily by specifying print styles as static properties of your components.